src/usr/isteps/istep10/host_proc_pcie_scominit.C

```diff
@@ -544,13 +549,13 @@ errlHndl_t computeProcPcieConfigAttrs(TARGETING::Target * i_pProcChipTarget)
             //On FSP based systems it has already been determined
             if (!INITSERVICE::spBaseServicesEnabled())
             {
-#ifdef DYNAMIC_BIFURCATION
+#if CONFIG_DYNAMIC_BIFURCATION || CONFIG_BARRELEYE_G2_DYNAMIC_BIFURCATION
```

Comment:
What is the motivation for a second config var if it triggers the same thing?

Reply:
It is the common code to apply the bifurcation attribute value. So the two config all need this code. And the two configs are exclusive. How about your opinion?

Reply:
The specific flag already depends on CONFIG_DYNAMIC_BIFURCATION being set.  So if this chunk of code is needed for all cases where that is set then just use CONFIG_DYNAMIC_BIFURCATION.  If it is needed for the G2 mode only then just use CONFIG_BARRELEYE_G2_DYNAMIC_BIFURCATION